polkadot/xcm/xcm-simulator/fuzzer/src/relay_chain.rs

```diff
@@ -259,3 +265,53 @@ construct_runtime!(
 		MessageQueue: pallet_message_queue,
 	}
 );
+
+#[sp_version::runtime_version]
```

Comment:
I don't see anywhere else in this PR that this runtime apis are needed. It seems like you just want to call into the methods exposed from `Executive`. Are you sure you need this?

Reply:
This was removed in 3281a1f.